src/solver/conv_mlir_igemm_wrw_xdlops.cpp

```diff
@@ -117,7 +117,6 @@ ConvSolution ConvMlirIgemmWrWXdlops::GetSolution(const ConvolutionContext& ctx)
 
     construction_parameters.kernel_name = GetKernelName();
     construction_parameters.kernel_file = construction_parameters.kernel_name + ".mlir";
-
```

Comment:
Hmm, again... It seems you really hate this empty line, right? :sunglasses:

Reply:
Haha I think I looked through the different solvers trying to find inconsistencies and apparently the empty line catch my attention :rofl: